tools/travis.bazelrc

```diff
@@ -0,0 +1,3 @@
+import %workspace%/.bazelrc
+build --disk_cache=/disk-cache/ --verbose_failures --spawn_strategy=sandboxed
+test --disk_cache=/disk-cache/ --spawn_strategy=sandboxed --nocache_test_results
```

Comment:
Can you add those noshow_progress flags back in too? Otherwise the travis logs end up with a _ton_ of noise.

Also nit: newline at EOF

Reply:
I removed the nolog intentionally since each build will be in it's own VM, there won't be a huge amount of text to scroll past.

Reply:
I think it's nice to see the build progress. 

I can go the other way, too.

Reply:
Up to you! Separating the bazel stuff out into its own stage definitely helps. You only have to scroll through the progress messages if it fails. (The reason I added that flag in the first place was that I was trying to see my sigh test failures, and I had to scroll through so much noise to get there)

That said, can you add --test_output=errors? I just looked at the Travis logs and you can't tell why the tests are failing.